lib/rules/no-plusplus.js

```diff
@@ -46,12 +46,28 @@ module.exports = {
                 if (allowInForAfterthought && node.parent.type === "ForStatement") {
                     return;
                 }
+
+                let fixerFunc;
+
+                if (node.parent.type !== "ReturnStatement") {
```

Comment:
should we fix for "ForStatement" as well ? otherwise this case won't be fixed:
`for (i = 0; i < l; i++) { console.log(i); }`